http2/testdata/common.proto

```
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+import "testdata/another/department.proto";
```

Buf (1.47.2)

import "testdata/another/department.proto": file does not exist

(COMPILE)